libs/assigner_runner/src/block_parser.cpp

```diff
@@ -0,0 +1,141 @@
+#include "zkevm_framework/assigner_runner/block_parser.hpp"
+
+#include <boost/algorithm/hex.hpp>
+#include <boost/endian.hpp>
+#include <boost/endian/conversion.hpp>
+#include <cstring>
+#include <fstream>
+#include <iostream>
+
+#include "block_schema.def"
+#include "zkevm_framework/json_helpers/json_helpers.hpp"
+
+template<typename T>
+std::optional<std::string> to_std_bytes(const boost::json::value &json_value, T &dst) noexcept {
```

Comment:
You have this in json_helpers already

Reply:
`json_helpers::to_std_bytes()` supports only `vector<std::byte>` as destination, but here we need `std::array `with different size